.devcontainer/codespaces-docs/welcome.rst

```diff
@@ -1,5 +1,60 @@
-Welcome to GitHub PyMAPDL Codespaces for Documentalist!!
-========================================================
 
-This codespace has been specifically set for documenting PyMAPDL.
+===========================================================
+Welcome to GitHub PyMAPDL Codespaces for Documentalist!! 📚
+===========================================================
 
+
+🛑 Disclaimer
+=============
+
+You **must use this Codespace only for developing, contributing, documenting and/or
+write examples for PyMAPDL**.
+
+Additionally, this Codespace is using an MAPDL Student version which is
+available `here <https://www.ansys.com/academic/students>`_
```

Comment:
```suggestion
Additionally, this Codespace is using an MAPDL Student version. For information on how Ansys
supports your learning Ansys software, see the Ansys for Students <https://www.ansys.com/academic/students>`_ page on the Ansys website.
```
Goggle dev style guide says to never use "here" for a link but the name of the title or page that the link takes you to.